modules/ghpp/module.json

```diff
@@ -20,7 +20,7 @@
 			"twitter": "@ForgeVTT"
 		}
 	],
-	"url": "https://www.dndbeyond.com/sources/ghpp",
+	"url": "https://www.dndbeyond.com/sources/dnd/ghpp",
```

Comment:
URL is valid, but redirects to `https://www.dndbeyond.com/sources/dnd/ghpp/grim-hollow-player-pack`